safe_extras/parameters/string_parameter.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, guid=None):
         :param guid: Optional unique identifier for this parameter. If none
             is specified one will be generated using python hash. This guid
             will be used when storing parameters in the registry.
-        :type guid: str
+        :type guid: str, None
         """
         super(StringParameter, self).__init__(guid)
-        self.expected_type = [str, unicode]
```

Comment:
Why do you should we not allow unicode here? I mean the if unicode is passed into it, then the type test would break. That is a condition that I had and it made sense to me to allow for unicode.